src/mlpack/tests/test_tools.hpp

```diff
@@ -48,6 +49,21 @@ inline void CheckMatrices(const arma::Mat<size_t>& a,
     BOOST_REQUIRE_EQUAL(a[i], b[i]);
 }
 
+template <typename FieldType,
+          typename = std::enable_if_t<
+              arma::is_Col<typename FieldType::object_type>::value ||
+              arma::is_Mat_only<typename FieldType::object_type>::value>>
+// Check the values of two field types
+inline void CheckFields(const FieldType& a,
+                          const FieldType& b)
```

Comment:
```suggestion
                        const FieldType& b)
```

Just a spacing fix. :)